.github/workflows/build-test.yml

```diff
@@ -1016,3 +1031,79 @@ jobs:
       # Alternative method for uploading
       # - name: Upload coverage to codecov
       #  run: bash <(curl -s https://codecov.io/bash) -f ./coverage/cobertura-coverage.xml -v
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    # This should happen in the release and insiders branch
+    if: github.repository == 'microsoft/vscode-jupyter' && (needs.pick_environment.outputs.release_channel == 'insider' || needs.pick_environment.outputs.release_channel == 'stable')
+    needs: [pick_environment, build-vsix, lint, ts_tests, smoke-tests]
+    environment:
+      name: ${{needs.pick_environment.outputs.release_channel}}
```

Comment:
Using Github Environments Releases
I have created two environments `insider` & `stable`.
For Insiders, they will get published directly.
For `stable` environment, this job will NOT run, until it is approved. there is a list of approvers that we can add (for now I had only added myself, to ensure this works properly, then we can add everyone else).

See here for more info:
https://docs.github.com/en/actions/reference/environments
https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments

Reply:
Oh yes, I will add @IanMatthewHuff  as he's going to be doing the release this time